commons-numbers-complex/src/test/java/org/apache/commons/numbers/complex/TestUtils.java

```diff
@@ -67,6 +68,19 @@ public static void assertSame(Complex expected, Complex actual) {
         Assertions.assertEquals(expected.getImaginary(), actual.getImaginary());
     }
 
+    /**
+     * Verifies that real and imaginary parts of the Complex and ComplexNumber arguments are
+     * exactly the same as defined by {@link Double#compare(double, double)}. Also
+     * ensures that NaN / infinite components match.
+     *
+     * @param expected the expected value
+     * @param actual the actual value
+     */
+    public static void assertSame(Complex expected, ComplexNumber actual) {
```

Comment:
Do we need this method? If so then add a message to the assertions for "real" and "imaginary"